content/en/docs/internal-documentation/incident-management.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Incident Management
+linkTitle: Incident Management
+---
+
+An incident refers to an event that can happen at any given time and
+ may cause a decrease in the quality or complete outage of one or
+ more of our services. Internal or external customers, our monitoring
+ and alerting systems, or a member of the SRE team can raise an incident.
```

Comment:
I would not redefine this information here as it may eventually conflict with the content on [The Source](https://source.redhat.com/groups/public/service-delivery/service_delivery_wiki/incident_management_process#jive_content_id_Incidents).

If a preamble is required I would just introduce the external documentation.